src/main/java/org/openpnp/machine/reference/ReferencePnpJobProcessor.java

```diff
@@ -141,7 +141,7 @@ public String toString() {
 
     protected Head head;
 
-    protected List<JobPlacement> jobPlacements = new ArrayList<>();
+    protected static List<JobPlacement> jobPlacements = new ArrayList<>();
```

Comment:
This, and the static methods added below should not be static and instead accessed via the interface.